src/debugger/nodeDebugWrapper.ts

```diff
@@ -43,6 +43,7 @@ interface ILaunchArgs {
     platform: string;
     target?: string;
     internalDebuggerPort?: any;

```

Comment:
I agree with the any. I'm saying that all the other user provided values should also be any, and they should be marked as string or number only after we've successfully parsed them or validate that they are the proper type.